ivy/functional/backends/torch/random.py

```diff
@@ -96,21 +96,18 @@ def randint(
     high: Union[int, torch.Tensor],
     /,
     *,
-    shape: Optional[Union[ivy.NativeShape, Sequence[int]]] = None,
+    shape: Optional[Union[ivy.NativeShape, Sequence[int]]] = (1,),
```

Comment:
This one also should be kept as `None`

Reply:
@hirwa-nshuti , thanks it's done. 

Why do you think `None` makes more sense in this case? 
Is it the convention for backend functions?

Reply:
It's because we have explicitly stated it in the functional API🙂
https://lets-unify.ai/ivy/functional/ivy/random/randint.html

Reply:
Thank you :)